backend/services/llm_providers/gemini_service.py

```
@@ -4,59 +4,59 @@
+                self.usage_metadata = None
+
+        class _DummyClient:
+            def __init__(self, service):
+                self.service = service
+            
+            def models(self):
+                class _DummyModels:
+                    def __init__(self, service):
+                        self.service = service
+                    
+                    def generate_content(self, model: str, contents: list):
```

Ruff (0.12.2)

Unused method argument: `model`

(ARG002)